recipes/python-stringcase/meta.yaml

```diff
@@ -0,0 +1,37 @@
+{% set name = "stringcase" %}
+{% set version = "1.2.0" %}
+
+package:
+  name: python-{{ name }}
```

Comment:
We should use the same name as pypi? Or is there a name conflict?

Reply:
renamed